### PR TITLE
Fix issue #844

### LINKED ISSUE #844: add feature to allow horizontal sections without countries/graticules
Necessary for proper contour layers.

### PR DESCRIPTION
**Pull request by joernu76**

### CODE REVIEW

**Comment by joernu76** [comment]
@Marilyth Hi May, could you try to reproduce this bug (the failing test is yours)? I cannot reproduce this or understand how the change impact the execution in any way.

File: mslib/mswms/mpl_hsec.py [hunk 1]
```diff
@@ -330,17 +331,18 @@ def plot_hsection(self, data, lats, lons, bbox=(-180, -90, 180, 90),
...
-        # scatter() for drawing the flight tracks and trajectories.
-        # Curiously, plot() works fine without this setting, but scatter()
-        # doesn't.
-        bm.fillcontinents(color='0.98', lake_color='white', zorder=0)
-        self._draw_auto_graticule(bm)
+        if self._plot_countries:
+            # Set up the map appearance.
+            bm.drawcoastlines(color='0.25')
+            bm.drawcountries(color='0.5')
+            bm.drawmapboundary(fill_color='white')
```
Comment (ReimarBauer) [inline mslib/mswms/mpl_hsec.py:338]: 'white' is now a constant, does this mean that the option from the Map Appeareance is disabled?

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
see my question to the UI colors

File: mslib/mswms/mpl_hsec.py (hunk 1 above)
Comment (joernu76) [reply mslib/mswms/mpl_hsec.py:338]: It was a constant before as well. No change here, I only made this code section optional in derived classes.

I think you confuse the client and server. The options affect only the client, this is server code.

**Review by ReimarBauer — APPROVED** [review]

**Pull request merged**